### PR TITLE
adding uat vpn profile generation

### PR DESCRIPTION
**Pull request by prashant3863**

### CODE REVIEW

**Comment by ajeygore** [comment]
I see hardcoded values and even internal IP address, please delete this PR. Please put configurations such as

- VPN Remote Hosts, and their values

Given these values, I should be able to add a remote host into Gate. And we should show drop downs for each profile. 

By default you will need enable users for each of these profiles. You can leverage groups to allow user to download specific profiles, we will list the options on home pages ONLY if user is part of that group. At least one host will have default access to all users.

So basically the way you do it is following

- Add a domain model call remote VPN hosts
- Add SSH authentication for each of the hosts
- Add VPN profile generation and copy command

Once you are done, then in generate VPN, you actually pull out this information and then execute appropriate commands.
